mbtiles/src/summary.rs

```diff
@@ -44,23 +45,28 @@ pub struct Summary {
 
 impl Display for Summary {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        writeln!(f, "Schema: {}", self.mbt_type)?;
+        writeln!(f, "{:15} {}", "File path:", self.file_path)?;
+        writeln!(f, "{:15} {}", "Schema:", self.mbt_type)?;
```

Comment:
I would prefer something like below, because:
- Your impl is a bit brittle towards ignoring newly added fields, my way we get a compilation warning
- I find it harder to read compared to spelling out where the spaces go manually

```suggestion
        let { file_path, mbt_type } = self;
        writeln!(f, "File path:     {file_path}")?;
        writeln!(f, "Schema:        {mbt_type}")?;
```

Reply:
I am already not printing all the available fields - only those I care about to print. JSON is the one that keeps it consistent.  Also, formatting with your approach is a bit less readable because it does not line up inside the `if` blocks.  Lastly, this approach is the same as used right below that - with the table formatting.  If we really want to clean that up (and I don't think it is worth it), we may want to use one of many table-formatting crates, rather than what I used.